package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A system that allows efficient, cost-effective management of our library.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "cross-env NODE_ENV=test nyc --reporter=html --reporter=text mocha --timeout=10000 --require @babel/register  --require @babel/polyfill",
```

Comment:
About the test command, we won't be needing `@babel/register  --require @babel/polyfill`. No code transformations have been planned at the moment.